dev-lang/bacon/bacon-4.7.2.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Basic to C converter"
+DESCRIPTION="The name 'BaCon' is an acronym for 'Basic Converter'. \
+It is a free BASIC to C translator for Unix-based systems, which \
+runs on most Unix/Linux/BSD platforms, including MacOSX. It intends \
+to be a programming aid in creating tools which can be compiled on \
+different platforms (both 32bit and 64bit), while trying to revive \
+the days of the good old BASIC."
+HOMEPAGE="https://chiselapp.com/user/bacon/repository/bacon/home"
+COPYRIGHT="2009-2023 Peter van Eerten"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://chiselapp.com/user/bacon/repository/bacon/attachdownload/bacon-$portVersion.tar.gz?page=Downloads&file=bacon-$portVersion.tar.gz"
+CHECKSUM_SHA256="619b721c820e1b11314cd7b88da39db3e76d9063605ee98aa35fb3fe635ab06c"
+
+ARCHITECTURES="all"
```

Comment:
This probably wont work on 32bit, libfltk is secondary arch only there, so needs secondaryArchSuffix and the likes here :)

Reply:
Ok, will check over the weekend as I am not yet familiar with that (and will amend the receipe accordingly).

Also got your messages on IRC:

- New version got published today, that is why I missed it. I was just unlucky with the release timing.
- Good point for the menu entry. Will also check on that over the weekend.